configure

```diff
@@ -2202,10 +2204,10 @@ check_higgstools_libs() {
         if [ "$extension" = "a" ] ; then
             if [ -z "$higgstools_lib_dir" ]; then
                 # assume that the linker will look in the default paths
-                HIGGSTOOLSLIBS="-Wl,-Bstatic -$HIGGSTOOLSLIBS_ -Wl,-Bdynamic"
+                HIGGSTOOLSLIBS="-Wl,-Bstatic -$HIGGSTOOLSLIBS_ model_specific/SM/libmodel_specific_SM.a -Wl,-Bdynamic"
```

Comment:
I think it would be better to write
~~~
HIGGSTOOLSLIBS="-Wl,-Bstatic -$HIGGSTOOLSLIBS_ model_specific/SM/libmodel_specific_SM.${extension} -Wl,-Bdynamic"
~~~
instead of the hard-coded `.a` extension

Reply:
Done in 97692f0677cd295dffcd1386dd95f9bff59b1388.